Makefile

```diff
@@ -23,22 +26,42 @@ test-postgresql:
 
 # Engine-specific test environments. In most cases you should run these with
 # `make -e ADDITIONAL_PYTEST_OPTIONS="--use-persistent-source-schema" test-<engine_type>`
+#
+# See: [Persistent Source Schema](/GLOSSARY.md#persistent-source-schema)
 .PHONY: test-bigquery
 test-bigquery:
 	hatch -v run bigquery-env:pytest -vv -n $(PARALLELISM) $(ADDITIONAL_PYTEST_OPTIONS) metricflow/test/
 
+.PHONY: populate-persistent-source-schema-bigquery
+populate-persistent-source-schema:
```

Comment:
add `-bigquery` suffix

Reply:
Nice catch. Apparently, if the name it wrong, `make` still succeeds with `nothing to do`